### DISCUSSION TITLE
Integrate flowise in intellij for custom code review

### DISCUSSION THREAD

**Discussion by tkondi** [FlowiseAI/Flowise#3431] [Q&A]
Has anyone successfully set up an AgentFlow execution in IntelliJ IDEA upon compilation? 
I'm looking to:
Pass the current file as read.
Have various workers inspect different aspects of the file.

I'm also interested in:
API availability for this process.
Suggestions on tools or frameworks to start with.

Any insights or starting points would be great!

**Comment by BigFoxMedia** [comment]
> Has anyone successfully set up an AgentFlow execution in IntelliJ IDEA upon compilation? 
> I'm looking to:
> Pass the current file as read.
> Have various workers inspect different aspects of the file.
> 
> I'm also interested in:
> API availability for this process.
> Suggestions on tools or frameworks to start with.
> 
> Any insights or starting points would be great!

Very interested in this topic but from a VSCode perspective. It's on my bucket list to attempt this but haven't gotten around to it yet.

**Reply by tkondi** [reply]
If you get a chance to work on it, please let me know.
Thanks
